test/index.js

```diff
@@ -273,31 +273,61 @@ describe('logger', () => {
         expect(this.output, 'to equal', fixtures[title]);
     });
 
-    it('should log an unhandled error', async function () {
-        const title = this.test.fullTitle();
-        const createLogger = this.createLogger(title);
+    describe('when an error is thrown', () => {
+        it('should log the error', async function () {
+            const title = this.test.fullTitle();
+            const createLogger = this.createLogger(title);
 
-        const logger = createLogger();
+            const logger = createLogger();
 
-        const context = {
-            method: 'GET',
-            originalUrl: '/'
-        };
+            const context = {
+                method: 'GET',
+                originalUrl: '/'
+            };
 
-        const next = () => {
-            const error = new Error();
-            error.stack = 'Error\n    at stack';
+            const next = () => {
+                const error = new Error();
+                error.stack = 'Error\n    at stack';
 
-            throw error;
-        };
+                throw error;
+            };
 
-        try {
-            await logger(context, next);
-        } catch (error) {
-            // Not interested in this one
-        }
+            try {
+                await logger(context, next);
+            } catch (error) {
+                // Not interested in this one
+            }
 
-        expect(this.output, 'to equal', fixtures[title]);
+            expect(this.output, 'to equal', fixtures[title]);
+        });
+
+        it('should log the error status if avaiable', async function () {
+            const title = this.test.fullTitle();
+            const createLogger = this.createLogger(title);
+
+            const logger = createLogger();
+
+            const context = {
+                method: 'GET',
+                originalUrl: '/'
+            };
+
+            const next = () => {
+                const error = new Error('GatewayTimeout');
```

Comment:
`ctx.throw` formats the error message as such. with an actual koa app, I had this in the logs:

```
──‣ ┈┈┈┈┈ GET┈ ┬ /
               ╎ ServiceUnavailableError: Yo
               ╎     at Object.throw (node_modules/koa/lib/context.js:91:23)
               ╎     at app.use (server.js:44:22)
               ╎     at dispatch (node_modules/koa-compose/index.js:44:32)
               ╎     at next (node_modules/koa-compose/index.js:45:18)
               ╎     at logger (/home/jmu/Projects/concurrency-logger/src/index.js:323:19)
               ╎     at dispatch (node_modules/koa-compose/index.js:44:32)
               ╎     at next (node_modules/koa-compose/index.js:45:18)
               ╎     at app.use (server.js:8:11)
               ╎     at dispatch (node_modules/koa-compose/index.js:44:32)
               ╎     at node_modules/koa-compose/index.js:36:12
503 ┈┈1ms GET┈ ┴ /
```

In this case `Yo` was my custom error message. Maybe I should put that into the fake error stack?